src/Logging/RequestErrorLogParameter.php

```diff
@@ -0,0 +1,98 @@
+<?php
+// phpcs:ignoreFile
```

Comment:
Any specific reason for ignoring phpcs and not adding doc comments?

Reply:
To avoid unnecessary boilerplate.

Reply:
We are using drupal standards here and this file can grow up to twice its size due to that reason, while this is only a parameter object. Drupal standard, however, has some unnecessary or unrefined restrictions which results in an antipattern.